Automattic/wp-calypso
# Generated design picker: Add sticky CTA button

#### Changes proposed in this Pull Request

This PR adds a footer to the generated design picker, which contains the "Continue" button. The footer functions as an alternative way users can access the CTA button without scrolling all the way back to the top. In addition, the footer can be sticky via component prop, so that the CTA button is always visible regardless of user scrolling.

CTA at top:
![Screen Shot 2022-05-11 at 3 23 56 PM](https://user-images.githubusercontent.com/797888/167792736-dd58c2f8-1ee7-4be8-a166-453bdd14f40a.png)

CTA at bottom (sticky):
![Screen Shot 2022-05-11 at 3 24 02 PM](https://user-images.githubusercontent.com/797888/167792785-20f9bef9-b04f-46da-a8b4-38137a1c65c9.png)

#### Testing instructions



* Make sure site vertical is set from the vertical question `/setup/vertical/siteSlug=${site_slug}`.
* Head to the site intent screen `/setup/intent?siteSlug=${site_slug}`, and click on the "Start building" button.
* Now, in the generated design picker, expect that once the CTA button is no longer in the viewport, the sticky footer will show up at the bottom of the viewport.
* Also, expect the sticky footer to not be visible on mobile resolution.



Related to #62237
